client/signup/config/flows.js

```diff
@@ -457,6 +457,10 @@ const Flows = {
 		return flows;
 	},
 
+	isValidFlow( flowName ) {
+		return Boolean( Flows.getFlows()[ flowName ] );
```

Comment:
We could use [`include()`](https://lodash.com/docs#includes) from Lodash instead here - which by the way is already imported in this file.

Reply:
`includes` looks at the value of the elements of the array, not at the existence of the key. Personally I'd rather use native JS than a library when it's only one simple line.